internal/cloud/awscloud/secure-instance.go

```diff
@@ -78,6 +78,11 @@ func (a *AWS) RunSecureInstance(iamProfile, keyName, cloudWatchGroup, hostname s
 		}
 	}()
 
+	previousSI, err := a.terminatePreviousSI(identity.InstanceID)
+	if err != nil {
+		logrus.Warningf("Unable to terminate previous secure instance with instance ID (%s): %v, this job is now likely to fail", previousSI, err)
+	}
```

Comment:
would it be also interesting as a metric to print the terminated `previousSI` in the `else` case too?
```suggestion
	} else {
	  logrus.Infof("Previous secure instance with instance ID (%s) terminated", previousSI)
	}
```

Reply:
sure